tests/expectations/both/constant.c

```diff
@@ -7,6 +7,14 @@
 
 #define FOO 10
 
+#define HEART '\u{2764}'
```

Comment:
So yeah, as test log on Travis currently says:

```
tests/expectations/constant.cpp:9:30: error: incomplete universal character name \u
 static const wchar_t HEART = '\u{2764}';
                              ^~~~~~~~~~
```

this will require some custom escaping.

Reply:
I changed the title to WIP because I am not sure what the correct way forward is.

Filtering the curly braces 
`value.value().escape_default().to_string().replace('{', "").replace('}', "")`
produces
`static const wchar_t HEART = '\u2764';` 
which leads to a warning 
```ignisvulpis@namenlos:~/development/cbindgen$ g++ -c /tmp/x.cpp 
/tmp/x.cpp:9:30: warning: multi-character character constant [-Wmultichar]
 static const wchar_t HEART = '\u2764';
```
Suggestions welcome.

Reply:
I think the correct output should be

```cpp
static const wchar_t HEART = L'\u2764';
```

Also note that `.replace('{', "").replace('}', "")` is not a great way to go about it, since now you are probably corrupting constants like `const c: char = '{';`.

Instead, you should check the character code and, if it's larger than 255, format the escape sequence manually.